test/save_load_scfres.jl

```diff
@@ -0,0 +1,25 @@
+using DFTK
+using WriteVTK
+using JLD2
+include("testcases.jl")
+
+Si = ElementPsp(14, psp=load_psp(silicon.psp))
+atoms = [Si => silicon.positions]
+model = model_LDA(silicon.lattice, atoms)
+kgrid = [1, 1, 1]
+Ecut = 7
+basis = PlaneWaveBasis(model, Ecut; kgrid = kgrid)
```

Comment:
For kwargs we don't put spaces around the equals, so
```suggestion
basis = PlaneWaveBasis(model, Ecut; kgrid=kgrid)
```